src/utilities/hypre_hopscotch_hash.h

```diff
@@ -581,9 +592,9 @@ void hypre_UnorderedIntMapDestroy( hypre_UnorderedIntMap *m );
 void hypre_UnorderedBigIntMapDestroy( hypre_UnorderedBigIntMap *m );
 
 // Query Operations .........................................................
-#ifdef HYPRE_CONCURRENT_HOPSCOTCH
-static inline HYPRE_Int hypre_UnorderedIntSetContains( hypre_UnorderedIntSet *s,
-                                                HYPRE_Int key )
+static inline HYPRE_Int
+hypre_UnorderedIntSetContains( hypre_UnorderedIntSet *s,
+                               HYPRE_Int              key )
```

Comment:
@rfalgout @victorapm This function only existed when `HYPRE_CONCURRENT_HOPSCOTCH` was defined. You now want to make it available as well even when it is not defined. These functions are complicated and I have not understood how it works. But from a high level, how can we guarantee it works, especially if changing it by commenting out the lines involving `segment` (see lines below with the `#ifdef`)?

Maybe you have dig into it so you understand how it works. If so, you can ignore my comments above.

Reply:
Hi Ruipeng, I haven't dug too much into the details of the implementation either but the same approach (commenting lines involving `segments`) is used in other functions and I tried to replicate that. Moreover, these changes don't break the regression tests.